packages/editor-ui/src/plugins/i18n/locales/en.json

```
@@ -681,6 +681,9 @@
 	"expressionModalInput.undefined": "[undefined]",
 	"expressionModalInput.null": "null",
 	"expressionTip.noExecutionData": "Execute previous nodes to use input data",
+	"expressionTip.typeDotPrimitive": "Type <code>.</code> for data transformation options",
+	"expressionTip.typeDotObject": "Type <code>.</code> for data transformation options, or to drill down",
```

1. Not sure if `drill down` will be understandable.

```suggestion
	"expressionTip.typeDotObject": "Type <code>.</code> for data transformation options, or to access a property",
```

2. Let's link to data transformation docs with `Learn more`?

---

Checking with david on this one